analyses/cellranger-analysis/README.md

```diff
@@ -0,0 +1,58 @@
+# Pipeline for running and summarizing Cell Ranger count for single or multiple libraries for sc-/sn-RNA-Seq Analysis in 10X Genomics data
+
+## Usage
+
+To run all the scripts in this module from the command line sequentially, use:
+
+```
+bsub < submit-multiple-jobs.sh
+```
+
+The `submit-multiple-jobs.sh` script is designed to be run as if it was called from this module directory even when called from outside of this directory. 
+   - Step 1: To run the `j1.sh` script to align single or multiple libraries in parallel, i.e., `run-cellranger-analysis`. 
+   - Step 2: To run `j2.sh` to summarize alignment results, i.e., `summarize-cellranger-analysis`. The latter script will be on hold and executed once all libraries are aligned and `j1.sh` is complete. This is been taken care of by `waiter.sh` script.
+
+Parameters according to the project and analysis strategy will need to be specified in the following scripts:
+- `j1.sh`: 
+   - `--file`: define path with `project_metadata` file. This is defined by default in the `./input/` dir within the `/sc-rna-seq-snap/analyses/cellranger-analysis` module. If the file lives in another folder, this path should be modified accordingly within the script. The file needs to be in `*.txt` file format. In addition, there is a code line to convert `*.tsv` file to `*.txt` file, if needed. User can specify the file path in there for the conversion. This file can contain either one or multiple samples as long as the following minimum parameters are provided: `ID`, `SAMPLE`, and `FASTQ` columns.
+   - `--transcriptome`: define reference genome to be used for the alignment. This points out to the path with the reference genome. There are 4 options accommodating for human, mouse, and dual index genomes: `GRCh38`, `mm10`, `GRCh38ANDmm10`, and `GRCh38_GFP_tdTomato`, respectively. If a different reference genome is required (not provided here), this can be added as an argument option in the `./util/run_cellranger.py` along with the path. Unless otherwise specified, the path with all reference genomes is by default and these are maintained by our team at the Bioinformatics core at DNB (./developmentalneurobiology/core_operations/Bioinformatics/common/ReferenceGenomes). 
```

Comment:
The mouse references being called for each of these options are technically GRCm39 (mm39), not mm10. See comments on python script. Perhaps these options should be updated here and in the python script to reflect that.

Reply:
```suggestion
   - `--transcriptome`: define reference genome to be used for the alignment. This points out to the path with the reference genome. There are 4 options accommodating for human, mouse, and dual index genomes: `GRCh38`, `GRCm39`, `GRCh38ANDGRCm39`, and `GRCh38_GFP_tdTomato`, respectively. If a different reference genome is required (not provided here), this can be added as an argument option in the `./util/run_cellranger.py` along with the path. Unless otherwise specified, the path with all reference genomes is by default and these are maintained by our team at the Bioinformatics core at DNB (./developmentalneurobiology/core_operations/Bioinformatics/common/ReferenceGenomes). 
```